packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -2235,6 +2332,42 @@ describe('lib/senders/emails:', () => {
     });
   }
 
+  for (const [type, test, opts = {}] of TESTS_3MONTHCOUPON) {
```

Comment:
Duplicating this logic won't be necessary if we keep using the `TESTS` array.

Reply:
fixed